recipes/tagbam/build.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -euo
+
+set -xe
+
+export CFLAGS="${CFLAGS} -fcommon"
+export CXXFLAGS="${CFLAGS} -fcommon"
+
+# Add workaround for SSH-based Git connections from Rust/cargo.  See https://github.com/rust-lang/cargo/issues/2078 for details.
+# We set CARGO_HOME because we don't pass on HOME to conda-build, thus rendering the default "${HOME}/.cargo" defunct.
+export CARGO_NET_GIT_FETCH_WITH_CLI=true CARGO_HOME="$(pwd)/.cargo"
+
+export BINDGEN_EXTRA_CLANG_ARGS="${CFLAGS} ${CPPFLAGS} ${LDFLAGS}"
+
+# build statically linked binary with Rust
```

Comment:
Please also add usage of cargo-bundle-licenses. See the docs for Rust recipes or other recipes as example.